queries/water.jinja2

```diff
@@ -220,8 +220,8 @@ FROM
   planet_osm_point t
 
 WHERE
-{% if mz_water_min_zoom >= 16 %}
-  mz_water_min_zoom IS NOT NULL
+{% if zoom >= 16 %}
+  mz_water_min_zoom IS NOT NULL AND
```

Comment:
Nice catch! :zap: I must have messed this up when making the changes throughout.